### ISSUE TITLE
Privacy issue

### ISSUE THREAD

**Issue by TomFreudenberg** [someteam/acha#13]
Hi, today I found post about acha and a project I am working on. When clicking at my icon, I saw my registered GitHub / Gravatar email address ?!

This is from my perspective a NO GO. I have disallowed to show my email on GitHub.

Seems to need a fix on that.

Tom

**Comment by ethercrow** [comment]
If you're uncomfortable your email address being visible, you shouldn't use it in your public commits. Currently anyone can see it by doing git clone && git log on your public repos.

**Comment by TomFreudenberg** [comment]
Yes you are right, but therefor someone HAS TO clone and check the log before getting the info. Using the Web from GitHub for example will not show that info. In case of your exposure to everyone - just a matter of time and google index - anyone will get the clear text adress infos. Your tool and idea wouldn't loose anything of it's pretty nifty idea when just linking to the GitHub usernames and not showing email-adresses. What do you think?

Btw. it's a pretty cool gimmick
Tom
